src/publicapi.js

```diff
@@ -158,6 +158,10 @@ function getInterface(v) {
       if (this.__controller.blurred) this.__controller.cursor.hide().parent.blur();
       return this;
     };
+    _.empty = _.clear = function() {
+      this.select().__controller.backspace();
+      return this;
+    };
```

Comment:
Actually another issue was: if you've got multiple \MathQuillMathFields but none of them are currently focused, which one do you programmatically write into? (Or affect with stuff like `.clear()`). 

So now we're remembering the last focused input (by adding a `.mq-last-focused` class)